.github/ISSUE_TEMPLATE.md

```diff
@@ -12,21 +12,19 @@
 
 #### Sonata packages
 
-```bash
```

Comment:
Looks like you messed up somewhere

Reply:
it is look like last commit with that changes was at Aug 8 by SonataCI

Reply:
You picked the wrong branch for this PR. You must pick `3.x` instead of the `master` branch

Reply:
@core23  at the top of the page you can see
nnmer  wants to merge 63 commits into sonata-project:**3.x** from nnmer:patch-config-yml-admin_services-param

so i don't think i miss the brach

Reply:
anyway, i will try to resubmit the PR again